content/docs/user-guide/assets/pipeline/_index.md

```diff
@@ -3,6 +3,7 @@ description:  Use the Open 3D Engine Asset Pipeline to convert your source art a
 linktitle: Asset pipeline
 title: Working with the Asset Pipeline and asset files
 weight: 100
+draft: true
```

Comment:
You'll need to remove the link to this section in `user-guide/assets/_index.md`.

Also, there are several other links into pipeline topics from other topics in the User Guide that will need to be updated to avoid broken links.

Reply:
Thanks, missed that one again. Taking this to `draft` might have uncovered a number of other areas where our documentation is incorrect as well so this might have some branching consequences.